regulations/tests/layers_formatting_tests.py

```diff
@@ -132,3 +132,36 @@ def ret_mock(arg):
         context = render.call_args[0][0]
         self.assertEqual(context['variable'], 'abc')
         self.assertEqual(context['subscript'], '123')
+
+
+    @patch('regulations.generator.layers.formatting.loader')
+    def test_apply_layer_dash(self, loader):
+        mocks = {'table': Mock(), 'note': Mock(), 'code': Mock(),
+                 'subscript': Mock(), 'dash': Mock()}
+
+        def ret_mock(arg):
+            for key in mocks:
+                if key in arg:
+                    return mocks[key]
+
+        loader.get_template.side_effect = ret_mock
+        render = mocks['dash'].render
```

Comment:
Minor, but these tests look super messy to me now. Can you think of a cute way of setting everything up for the four tests more concisely?